packages/mesos/build

```diff
@@ -17,8 +17,7 @@ LIBS="-lssl -lcrypto" LDFLAGS="-L/opt/mesosphere/active/openssl/lib" "/pkg/src/m
   --with-libevent=/opt/mesosphere/active/libevent \
   --with-curl=/opt/mesosphere/active/curl \
   --sbindir="$PKG_PATH/bin"
-# TODO(cmaloney): -j8 is almost certainly wrong.
-make -j8
```

Comment:
`-j` without arguments actually results in `-j<infinity>` which is more problematic typically (And makes things go slower than doing the "right" number of things.


From the man page:
Specifies the number of jobs (commands) to run simultaneously. If there is more than one -j option, the last one is effective. If the -j option is given without an argument, make will not limit the number of jobs that can run simultaneously.